main.go

```diff
@@ -17,11 +16,29 @@ import (
 )
 
 func main() {
-	startRun(os.Stdin, &request.DefaultRequestHandler{})
+	var mqttURL string
+	flag.StringVar(&mqttURL, "mqtturl", "", "MQTTURL")
```

Comment:
@bzwei we wont be running from the command line parameters anymore, instead we would have a config file with all the parameters from well known location like /etc/redhat_mqtt_client/catalog.conf

I have started this file which we can start with https://github.com/mkanoor/python_catalog_mqtt_client/blob/master/config/sample.conf

We would have to redo this area again so lets switch to the conf file.

Reply:
sounds good. should we have a default location of the conf file and overridable from command line args?